creator-node/src/utils.js

```diff
@@ -78,24 +87,27 @@ async function getIPFSPeerId (ipfs, config) {
  * Cat single byte of file at given filepath. If ipfs.cat() call takes longer than the timeout time or
  * something goes wrong, an error will be thrown.
 */
-const ipfsSingleByteCat = (path, req, timeout = 1000) => new Promise(async (resolve, reject) => {
-  const start = Date.now()
-  let ipfs = req.app.get('ipfsLatestAPI')
+const ipfsSingleByteCat = (path, logContext, timeout = 1000) => {
```

Comment:
i don't think all instances of this have been modified to use log context like routes/files.js line 111 and 190

Reply:
good catch! will fix